app/src/main/java/br/com/nespressohelper/daitan/nespressohelper/BackEndCommHandler.java

```diff
@@ -0,0 +1,15 @@
+package br.com.nespressohelper.daitan.nespressohelper;
+
+/**
+ * Created by estrela on 12/27/17.
+ */
+
+public interface BackEndCommHandler {
+
+    enum Method{GET, POST}
+
+    void handleResponse(String response, Method m);
+    void handleResponse(int code, Method m);
```

Comment:
Please refrain from using single letter named variables. They are really hard to keep track of.